.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "submodules/seqan3"]
 	path = submodules/seqan3
 	url = ../../seqan/seqan3.git
+[submodule "submodules/fmindex-collection"]
+	path = submodules/fmindex-collection
+	url = ../../fmindex-collection.git
```

Comment:
```suggestion
	url = ../../SGSSGene/fmindex-collection.git
```

Reply:
Sorry, I had this wrong initially. I immediately corrected in my comment, but it seems you were too fast in copying the old code.